src/Themes/VuetifyTheme/Recipes/GridSystem.recipe.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "Grid System Shortcodes",
```

Comment:
Possibly needs to mention that this a Vuetify Grid System as the templates use vuetify components.